WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/BottomSheetNoticeView.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
+struct BottomSheetNoticeView: View {
```

Comment:
Nice generic view! Perhaps we can make the name more generic, after all we can use it in scenarios other than bottom and sheet. Something like `DismissableNoticeView` ?

Reply:
Definitely, done: ad4287c2a805da29e0e1170b277214407e1ee09d